### PR TITLE
admins but they get to make voice announcements too

### PR DESCRIPTION
**Pull request by Bibby0110**
?

### CODE REVIEW

**Comment by alexkar598** [comment]
Ah yes I love to hear admin voice while getting abused with supply pods

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -162,7 +162,7 @@
 				return
 			make_announcement(usr)
 		if ("makeVoiceAnnouncement")
-			if (!authenticated_as_non_silicon_captain(usr))
+			if (!authenticated_as_non_silicon_captain(usr) || !IsAdminGhost(usr))
```
Comment (adamsong) [inline code/game/machinery/computer/communications.dm:165]: Not necessary, admin ghosts authenticate as non silicon captains.

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -390,7 +390,7 @@
 
 					data["alertLevelTick"] = alert_level_tick
 					data["canMakeAnnouncement"] = TRUE
-					data["canMakeVoiceAnnouncement"] = ishuman(user)
+					data["canMakeVoiceAnnouncement"] = ishuman(user) || IsAdminGhost(user)
```
Comment (adamsong) [inline code/game/machinery/computer/communications.dm:393]: Why not move this up to the authenticated_as_non_silicon_captain block, and set it to true?

**Review by adamsong — CHANGES_REQUESTED** [review]
See two previous comments

**Pull request closed without merging**